Pod/Classes/PhxPush.m

```diff
@@ -76,7 +74,7 @@ - (PhxPush*)onReceive:(NSString *)status callback:(OnMessage)callback {
     return self;
 }
 
-- (PhxPush*)after:(int)ms callback:(After)callback {
+- (PhxPush*)after:(NSTimeInterval)ms callback:(After)callback {
```

Comment:
ok, looking at the implementation it seems like the `ms` parameter was always used like a time-interval, so actually declaring it like one seems to be fine...

In that case, I think it makes sense to also rename the parameter to avoid the confusion with `milliseconds`.